cllc-public-app/ClientApp/src/app/components/applications/tied-house-decleration/tied-house-decleration-form/tied-house-declaration-form.component.ts

```diff
@@ -133,7 +137,7 @@ export class TiedHouseDeclarationFormComponent extends FormBase implements OnIni
   save() {
     this.form.markAllAsTouched();
 
-    if (!this.form.valid) {
+    if (!this.form.valid || this.form.get('associatedLiquorLicense')?.value.length ==0 ) {
```

Comment:
Is the `associatedLiquorLicense` check here necessary, given the form is already using the `requiredFormArray` validator?